src/group_theory/solvable.lean

```diff
@@ -175,4 +227,42 @@ begin
   ... = ⊥ : rfl,
 end
 
+lemma is_solvable_of_top_eq_bot (h : (⊤ : subgroup G) = ⊥) : is_solvable G :=
+⟨⟨0, by simp *⟩⟩
+
+lemma is_solvable_of_subsingleton [subsingleton G] : is_solvable G :=
```

Comment:
This can be an `instance` instead of `lemma`, because the type class system can figure out all the assumptions on its own. In that sense, this declaration allows for more automation than the `top_eq_bot` version. But I'm happy with having both.

Reply:
Okay, I changed it to an instance. The linter told me that I have to make its priority lower than 1000.